x-pack/plugins/reporting/server/plugin.ts

```diff
@@ -88,6 +94,10 @@ export class ReportingPlugin
 
       this.logger.debug('Start complete');
       this.start$.next(true);
+    })().catch((e) => {
+      this.logger.error(
+        `Error in Reporting startup, reporting may not function properly\n` + e.stack
+      );
```

Comment:
`logger.error` does a good job of logging an error when it receives one as a parameter. How about:
```
this.logger.error(`Error in Reporting startup, reporting may not function properly.`);
this.logger.error(e);
```

Reply:
nice, TIL!